pytest_pootle/fixtures/models/user.py

```diff
@@ -12,6 +12,18 @@
 from pytest_pootle.env import TEST_USERS
 
 
+@pytest.fixture(
+    scope="session",
+    params=["nobody", "admin", "member", "member2"])
+def request_users(request):
+    from django.contrib.auth import get_user_model
```

Comment:
Then it is just safer to leave it where it is. A bunch of other fixtures in this module also have imports.